### DISCUSSION TITLE
Steps Color

### DISCUSSION THREAD

**Discussion by Ettorebigm** [saadeghi/daisyui#1947] [Q&A] [answered]
Hi
How do I change the background of the line between the two steps?
![Schermata 2023-06-07 alle 23 55 11](https://github.com/saadeghi/daisyui/assets/134105000/a12739f6-3f62-4555-bbc1-74c70bb4fdf9)

Kind regards

**Comment by saadeghi** [comment, marked as answer]
By default, when you use a color like `step-primary` the current step and the line before it will get the color.  
But if you only want to color the line, you can use a class name like `before:!bg-accent` OR `[.steps>&]:before:bg-primary`

https://play.tailwindcss.com/FIWM98lidw
